packages/react/src/Form/core/getOptionClassName.ts

```diff
@@ -10,7 +10,7 @@ export function getOptionClassName(
     classModifier,
     disabled ? "disabled" : undefined,
   ]
-    .filter((c) => !!c)
+    .filter((c) => Boolean(c))
```

Comment:
On ne peut pas juste faire ça ?
```suggestion
    .filter(Boolean)
```

Reply:
En effet ça a l'air de fonctionner 👍